app/tests/e2e/test_wallet_credentials.py

```diff
@@ -60,3 +62,42 @@ async def test_get_and_delete_credential_record(
             f"{WALLET_CREDENTIALS_PATH}/{credential_id}"
         )
     assert exc.value.status_code == 404
+
+
+@pytest.mark.anyio
+@pytest.mark.skipif(
+    TestMode.regression_run in TestMode.fixture_params,
+    reason="Skipping due to regression run",
+)
+@pytest.mark.parametrize("issue_alice_many_creds", [3], indirect=True)
+async def test_get_credential_record_with_limit(
+    alice_member_client: RichAsyncClient,
+    issue_alice_many_creds: List[CredentialExchange],  # pylint: disable=unused-argument
+):
+    valid_params = [
+        {"limit": 1},
+        {"limit": 2},
+        {"limit": 3},
+        {"limit": 4},
+        {"limit": 1, "offset": 4},
```

Comment:
Just add one more for completeness:
`{"limit": 2, "offset": 2}`

expected length for that = 1